crates/re_viewport/src/space_view.rs

```diff
@@ -84,6 +73,47 @@ impl SpaceViewBlueprint {
         }
     }
 
+    /// Placeholder name displayed in the UI if the user hasn't explicitly named the space view.
+    #[allow(clippy::unused_self)]
+    pub fn missing_name_placeholder(&self) -> String {
+        let entity_path = self
+            .space_origin
+            .as_slice()
```

Comment:
The `.as_slice` shouldn't be needed

Reply:
Subtle: it _was_ needed because `EntityPath::iter()` returned `impl Iterator<Item = _>`, which is not compatible with `.rev()`. Changing the return type to `impl DoubleEndedIterator<Item = _>` made it work.